content/docs/pipelines/overview/concepts/pipeline.md

```diff
@@ -5,17 +5,16 @@ weight = 10
 +++
 
 A *pipeline* is a description of a machine learning (ML) workflow, including all
-of the components in the workflow and how the components relate to each other in
+of the [components](/docs/pipelines/overview/concepts/component/) in the workflow and how the components relate to each other in
 the form of a [graph](/docs/pipelines/concepts/graph/). The pipeline
 configuration includes the definition of the inputs (parameters) required to run
 the pipeline and the inputs and outputs of each component.
 
 When you run a pipeline, the system launches one or more Kubernetes Pods
-corresponding to the steps (components) in your workflow (pipeline). The Pods
+corresponding to the [steps (components)](/docs/pipelines/overview/concepts/step/) in your workflow (pipeline). The Pods
 start Docker containers, and the containers in turn start your programs.
 
-After developing your pipeline, you can upload and share it on the 
-Kubeflow Pipelines UI.
+After developing your pipeline, you can upload using the Kubeflow Pipelines UI or the Kubeflow Pipelines SDK.
```

Comment:
Suggested revision: 

After developing your pipeline, you can upload your pipeline using the Kubeflow Pipelines UI or the Kubeflow Pipelines SDK.

Reply:
Thanks @joeliedtke for reviewing this PR. I have modified the file as what you suggested.